File: CodeFestival/Q31-Q40/Q40_놀이동산에_가자.js
const limit = prompt("몸무게 제한을 입력해주세요.");
const n = prompt("몇 명인지 입력해주세요.");
let member = [];
let weight = 0;
let count = 0;
for (let i=0; i<n; i++) {
    member.push(parseInt(prompt("몸무게를 입력해주세요."),10));
}

for (i=0; i<n; i++) {
    weight += member[i];
    if (weight <= limit) {
        count += 1;
    }
}

console.log(count);

// let total = 0;
// let count = 0;
// const limit = prompt('제한 무게를 입력하세요.');
// const n = prompt('인원수를 입력하세요.');

// for (let i=1; i<=n; i++){
//   total += parseInt(prompt('무게를 입력해주세요.'), 10);
//   if (total <= limit){
// 		count = i;
//   }
// }

// console.log(count);

File: CodeFestival/Q31-Q40/Q40_놀이동산에_가자.py
limit = int(input())
n = int(input())
member = []
count = 0
weight = 0
for i in range(0, n, 1):
    member.append(int(input()))

# member.sort()
for i in member:
    weight += i
    if (weight <= limit):
        count += 1

print(count)

# 다른 답
# total = 0
# count = 0
# limit = int(input())
# n = int(input())
 
# for i in range(n):
#     if total <= limit:
#         total += int(input())
#         count = i
# print(count)
